nengo_gui/components/__init__.py

```diff
@@ -1,9 +1,21 @@
-from .slider import Slider, SliderTemplate
-from .value import Value, ValueTemplate
-from .xyvalue import XYValue, XYValueTemplate
-from .sim_control import SimControl, SimControlTemplate
-from .raster import Raster, RasterTemplate
-from .voltage import Voltage, VoltageTemplate
-from .pointer import Pointer, PointerTemplate
-from .netgraph import NetGraph, NetGraphTemplate
-from .ace_editor import AceEditor, AceEditorTemplate
+import inspect
+import sys
+
+from .component import Component
+from .slider import Slider
+from .value import Value
+from .xyvalue import XYValue
+from .sim_control import SimControl
+from .raster import Raster
+from .voltage import Voltage
+from .pointer import Pointer
+from .netgraph import NetGraph
+from .ace_editor import AceEditor
+
+# Old versions of the .cfg files used Templates which had slightly different
+# names than the Components currently usef.  This code is needed to
+# successfully parse those old .cfg files
+this_module = sys.modules[__name__]
```

Comment:
I know this saves us a couple of lines, but this seems like a weird place to put in the mental effort to read/write this Python magic and realize what's going on. If I'm reading this all correctly, you could replace it with
```python
SliderTemplate = Slider
ValueTemplate = Value
XYValueTemplate = XYValue
# and so on
```
which is 9 lines versus these 4, but it definitely takes less time to read and understand the 9 line version. There's also no reason to bother doing this for new components, so it doesn't matter if it gets out of date. Ideally, we'll remove the 9 lines eventually anyway.